_posts/2015-03-stitch-fix-heart-unix.md

```diff
@@ -0,0 +1,139 @@
+*Proposed Outline:*
+```
+I. Introduction
+II. Table of contents by problem
+III. Each problem
+  A. Title
+    1. Attribution and environment sub-heading
+  B. Problem
+  C. Solution
+    1. Input
+    2. Command
+    3. Explanation
+IV. ~~Closing~~?
+```
+
+--
+
+I &#10084; UNIX and using the command line; they help me solve problems at Stitch Fix.  I'm not alone.  Across the Data Science and Engineering teams, we're constantly solving problems with UNIX and the command line.  **TODO: Closing sentence replaces closing section?**  
+
+   
+## Problems and Solutions
+ - Files
+   - [What size are these images?](#check-image-size)
+   - [What files recently changed?]()
+   - [How many lines are in this data dump?]()
+   - [How do you bulk change filenames?]()
+   - [Can I efficiently and securely copy these files?]()
```

Comment:
OK, I'll continue. 

Do you think the Q&A concept works for interlinking from Table of Contents? 

1. "What size are these images?" -> "Check Image Size"
1. "What files recently changed?" -> "Find Recently Changed Files"